examples/webgpu_tsl_editor.html

```diff
@@ -158,11 +154,14 @@
 
 					const showCode = () => {
 
-						result.setValue( nodeBuilder[ options.shader + 'Shader' ] );
+						result.setValue( rawShader[ options.shader + 'Shader' ] );
 						result.revealLine( 1 );
 
 					};
 
+					const webGLRenderer = new THREE.WebGPURenderer( { forceWebGL: true } );
+					webGLRenderer.init();
```

Comment:
I've implemented it like suggested in https://github.com/mrdoob/three.js/issues/28984#issuecomment-2253318912, but there is one problem. If WebGPU isn't available, you can't force a WGSL output for debugging purposes.

Maybe `getRawShader()` would behave more reliable if you can request the output with a parameter. In this way, the method produces the same result no matter on what system it is executed.

Reply:
We could added `getRawShaderAsync()` too.